examples/analytics-vendors.amp.html

```diff
@@ -83,6 +83,7 @@
       <option>piano</option>
       <option>quantcast</option>
       <option>rakam</option>
+      <option>scroll</option>
```

Comment:
Given that you don't want any request to be sent without your amp-access scroll service, I think it doesn't make sense to include it in this example. Let's remove it : )